examples/tennis/Tennis.cpp

```diff
@@ -200,8 +200,7 @@ int main()
             }
 
             // Move the ball
-            float factor = ballSpeed * deltaTime;
-            ball.move({std::cos(ballAngle.asRadians()) * factor, std::sin(ballAngle.asRadians()) * factor});
+            ball.move({ballSpeed * deltaTime, ballAngle});
```

Comment:
Nice one! This also shows the strength of a type-safe `sf::Angle` type, otherwise we would have needed something like `sf::Vector2f::fromPolar(r, phi)`.

Are people generally OK with this being a constructor, i.e. allowing short syntax?